### DISCUSSION TITLE
Only outerwall with half height.

### DISCUSSION THREAD

**Discussion by lohiaprateek** [smartavionics/Cura#254] [Ideas]
Hi, Just came across this fork and loving it. Ive been a Cura user since they skinned skeinforge in 2012 but soon migrated to Kisslicer as it was one of the fastest slicers out there in 2012 and then just got used to it. But I guess I need to move back to Cura specifically this fork and its implementation of Vase Mode.

One feature is miss from the old Cura 12.12A was the Duplicate outline feature. It essentially just prints the outermost wall twice with half the layer height so as to improve visual looks. Every Slicer only allows you to change after how many layers one can print infill, this achieving kind of simillar results. 

Things add up when you have 4-5 line thickness on outerwall. Then it tends to take lot of time. Why this feature was removed in place of the not so efficient version I dont know, but can this be added in this fork. Attaching the screenshot of the feature and the write up.

![Dup Outline Cura 12 12A](https://github.com/smartavionics/Cura/assets/4669269/1c72daf3-5b98-48f8-9339-35058effeeb0)

**Comment by smartavionics** [comment]
Hi @lohiaprateek. I have tried implementing this. Please zip up a sample project and attach to this discussion and I will slice it and post the gcode for you to evaluate.

**Reply by lohiaprateek** [reply]
I dont have any project at hand. But we could try with a test cylinder?

Also you may download old Cura 12.12A from the link to see exports of gcode, I couldn't upload it here due to size restrictions. The zip also contains a cylinder STL.

https://1drv.ms/u/s!Aixf9Gp8GftHoI5qZ8LDoESf30EOXw?e=Of4Vst

**Comment by lohiaprateek** [comment]
Hi, thank you, will do a test print today and revert. Thank you once again

**Reply by lohiaprateek** [reply]
Works as planned, thank you
